neqo-qpack/src/encoder.rs

```diff
@@ -348,20 +348,24 @@ impl QPackEncoder {
         conn: &mut Connection,
         h: &[Header],
         stream_id: StreamId,
-    ) -> Res<HeaderEncoder> {
+    ) -> HeaderEncoder {
         qdebug!([self], "encoding headers.");
 
         let mut encoder_blocked = false;
         // Try to send capacity instructions if present.
         match self.send(conn) {
```

Comment:
Also, I think that you want to rename this "send" function to something more appropriate: update_encoder_stream() or send_encoder_updates() spring to mind as options.  I like terse names, but this is a little too terse.